docs/sphinx/sections/api/apidocs/libraries/dagster-airbyte.rst

```diff
@@ -0,0 +1,21 @@
+Airbyte (dagster-airbyte)
+---------------------------
+
+This library provides a Dagster integration with `Airbyte <https://www.airbyte.com/>`_.
+
+.. currentmodule:: dagster_airbyte
+
+Ops
```

Comment:
would be nice to also document the `AirbyteOutput` type

Reply:
Have one example how to this is made?

Reply:
https://raw.githubusercontent.com/dagster-io/dagster/master/docs/sphinx/sections/api/apidocs/libraries/dagster-dbt.rst in the "Types" section here (honestly the Fivetran docs should document the FivetranOutput type as well).